plugin.video.greekvoice/resources/lib/scrapetube/wrapper.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
```

Comment:
No need for file encoding declaration in Python 3. It's ignored.

Reply:
Done